provisioning/example.yaml

```diff
@@ -6,11 +6,19 @@ apiVersion: 1
 
```

Comment:
@priya-ni we gave some initial feedback on the Draft PR. Please let us know if that was the type of feedback you were looking for when creating the draft or if you have questions that were not addressed.